ParraCore/Logger/ParraWrappedLogMessage.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  ParraWrappedLogMessage.swift
+//  ParraCore
+//
+//  Created by Mick MacCallum on 6/24/23.
+//  Copyright © 2023 Parra, Inc. All rights reserved.
+//
+
+import Foundation
+
+public enum ParraWrappedLogMessage {
```

Comment:
Same thing with the protocol here. Could have an overarching protocol that ParraError and String conform to and then people using it can make custom conformances for their own error types or event for random objects

Reply:
@ianmaccallum so that is what this is basically what this is intended to be. For basic messages resolving to String is fine and any of a user's errors will conform to Error and be handled by this. At the top level this needs to stay an enum because I need to switch over all the cases and enforce that the associated objects are functions that return the actual log values, since this is used somewhere that we're using a lot of auto closures.